frontend/sass/_header.scss

```diff
@@ -1,60 +1,60 @@
 header {
   background-color: $header-bg-color;
-}
 
-$nav-padding: .45em;
+  $nav-padding: .45em;
```

Comment:
nested all these inside of `header` since I didn't want this styling to necessarily apply to the `nav` element in `SitePublishedFilesTable`

Reply:
this was when I thought I could actually page to random pages of results and would need a fuller paging implementation. perhaps best to revert this change and remove the `nav` element that holds the "Previous" and "Next" buttons from `SitePublishedFilesTable`. Though those buttons are semantically a form of navigation. hrm...

Reply:
The buttons can definitely stay wrapped inside a nav, is it possible to not tie the `nav` styles directly to the tag? The styles controlling the buttons on the `SitePublishedFilesTable` should probably live in a class anyway.

Reply:
Yeah, I was just going for minimal changes to the existing `nav` styling. That element was previously only used in our main site `header` element.